### DISCUSSION TITLE
Badges are getting kind of big

### DISCUSSION THREAD

**Discussion by mattmassicotte** [SwiftPackageIndex/SwiftPackageIndex-Server#2862] [Ideas]
With visionOS now supported, and (hopefully!) catalyst on the way, many of my package's badges no longer fit on a iPhone mini screen comfortably.

So I got to thinking, is "Platform Compatibility" actually even the best wording? Same goes for "Swift Compatibility". I think there is zero ambiguity by dropping the word "Compatibility" and it would make for a much tidier badge.

"Platforms"
"Swift"

What do you think?

**Comment by finestructure** [comment]
That's a good idea, I'd be on board with that!

**Comment by daveverwer** [comment]
Totally agree. Tracked (and fix incoming) in #2863
